src/services/routes/org.routes.ts

```diff
@@ -52,5 +65,41 @@ export default function registerRoutes(
     },
   });
 
+  fastify.route({
+    method: ['PATCH'],
+    url: '/orgs/:orgName',
+
+    schema: {
+      params: ORG_ROUTE_PARAMS,
+      body: ORG_SCHEMA_PATCH,
+    },
+
+    async handler(request, reply): Promise<void> {
+      const { orgName } = request.params;
+      const serviceOptions = {
+        logger: this.log,
+        dbConnection: this.mongoose,
+      };
+
+      const getOrgResult = await getOrg(orgName, serviceOptions);
+      if (!getOrgResult.didSucceed) {
+        await reply.code(RESPONSE_CODE_BY_PROBLEM[getOrgResult.reason]).send({
+          type: getOrgResult.reason,
+        });
+        return;
+      }
+
+      const result = await updateOrg(orgName, request.body, serviceOptions);
+      if (result.didSucceed) {
+        await reply.code(HTTP_STATUS_CODES.OK).send();
```

Comment:
I think this is ambiguous from a code perspective, and _maybe_ invalid as well. What's the response here? `text/plain` with an empty body? `application/json` with an empty body? The latter would be invalid JSON.

I think returning nothing is perfectly acceptable, but in that case we should return a `204` response.